app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "Help") %>
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <%= link_to "Rails Tutorial help section", "http://www.railstutorial.org/help" %>.
+  To get help on this sample app, see the
+  <%= link_to "", "http://www.railstutorial.org/book" %>
+  <%= link_to "http://www.railstutorial.org/book" do %>
+  <%= content_tag :em, "Ruby on Rails Tutorial." %>
```

Comment:
Please use I18n and 2 space indentation inside a block

Reply:
done